cmd/stackmargin/src/lib.rs

```diff
@@ -47,10 +47,18 @@ fn stackmargin(context: &mut humility::ExecutionContext) -> Result<()> {
     let (base, size) = hubris.task_table(core)?;
     let task = hubris.lookup_struct_byname("Task")?;
     let taskdesc = hubris.lookup_struct_byname("TaskDesc")?;
+    let dump_task = hubris.dump_task();
 
     let mut taskblock: Vec<u8> = vec![];
     taskblock.resize_with(task.size * size as usize, Default::default);
-    core.read_8(base, taskblock.as_mut_slice())?;
+
+    if let Some(HubrisTask::Task(i)) = dump_task {
```

Comment:
I don't understand the changes in this file; it looks like we're skipping reading the task block for the dump task?

Reply:
I think the nomenclature here is confusing; changing this from `dump_task` to `task_dump` and adding a comment.

Reply:
Aha, I now understand: this is a special case for if we're operating on a dump and the dump only contains a single task.  A comment to that effect would be welcome!

Reply:
I think the nomenclature change has made this much clearer, but let me know if you think it should be clearer still.